tests/mbed/enc/CMakeLists.txt

```diff
@@ -22,9 +22,23 @@ function(add_mbed_test_enclave NAME)
 
   set(MBEDTLS_TESTS_DIR "${PROJECT_SOURCE_DIR}/3rdparty/mbedtls/mbedtls/tests")
 
+  if (WIN32)
+     find_program(GIT git)
+     get_filename_component(GIT_DIR ${GIT} DIRECTORY)
+     find_program(PERL perl
+        PATHS "C:/Program Files/Git/usr/bin" "${GIT_DIR}/../usr/bin"
+        NO_DEFAULT_PATH)
```

Comment:
I think it's fine for it to look in the default paths, so we can remove this.

Reply:
@andschwa -- The problem is that we do not add the directory C:/Program Files/Git/usr/bin to our path and so by default, no Perl will be found on Windows.

Reply:
This is the primary reason for making the change as no Perl exists in any of the default paths on our Windows VMs. Hence, {PERL_EXECUTABLE} is a blank on Windows.

Reply:
This code _only_ looks in the two given paths because NO_DEFAULT_PATH was specified. Please remove that so that the PATH environment variable is also checked.

Reply:
oh, ok. Thanks for clarifying. Ignore my changes in flight.